bench/bench.go

```diff
@@ -254,16 +254,16 @@ func (bm *Benchmark) Report() string {
 	var buffer bytes.Buffer
 
 	indent := ""
+	if !bm.Pubs.HasSamples() && !bm.Subs.HasSamples() {
+		return "No publisher or subscribers. Nothing to report."
```

Comment:
yes if the tool didn't check the args, and simply passes them. This will catch it. In the case the client code didn't perform the check, the op still runs, and the message at least will let them know.